messagediff.go

```diff
@@ -152,21 +152,27 @@ func (d *Diff) diff(aVal, bVal reflect.Value, path Path) bool {
 		}
 	case reflect.Struct:
 		typ := aVal.Type()
+	FIELDS_LOOP:
 		for i := 0; i < typ.NumField(); i++ {
 			index := []int{i}
 			field := typ.FieldByIndex(index)
 			if field.Tag.Get("testdiff") == "ignore" { // skip fields marked to be ignored
 				continue
 			}
+			for _, o := range options {
+				if skip := o.apply(&opts{field: field}); skip {
+					break FIELDS_LOOP
```

Comment:
I feel like this should be `continue` not `break` since break will stop diffing all fields after the skip one.